### ISSUE TITLE
Consider reducing dependencies

### ISSUE THREAD

**Issue by DannyBen** [Zelaf/uptime-kuma-service-push#13]
I believe that these can be removed, as they are available everywhere no?

- tail 
- cat 
- echo 
- printf 
- mkdir 

https://github.com/Zelaf/uptime-kuma-service-push/blob/b553772e888f81060c3023656858da6e32b695db/bashly.yml#L7-L16

**Comment by Zelaf** [comment]
Figured why not have them just in case since they're used by Bashly and some libraries as well as my implementations. No harm really, question would be if the saves in removing redundancy is worth it. 

I tried to look into what's most available for different Linux systems but in the end I'm still mainly configuring this for debian-like systems. So would probably be best to remove it anyway in terms of having the bashly.yml shorter.

What I've found so far is that `echo`'s `-e` implementation is a *bit* non-standard so I've tried to use `printf` as much as possible while taking inspiration from the Bashly generated script. `mkdir` I have very rarely encountered not existing but that's mostly in docker systems or non-standard embedded Linux installations.

But for the sake of easier `bashly.yml` I think I will.
